src/components/card/FlipCard.js

```diff
@@ -0,0 +1,140 @@
+import styled, { css } from 'styled-components/macro'
+import { useState } from 'react'
+import mainBackground from '../../assets/images/mainBackground.jpg'
+
+export default function FlipCard(props) {
+  const [isCardFlipped, setIsCardFlipped] = useState(false)
+  const isFreeOfCharge = props.freeOfCharge
+  const handleFlipCardClick = () => {
+    setIsCardFlipped(!isCardFlipped)
+  }
+  return (
+    <Card onClick={handleFlipCardClick}>
+      <CardContent isCardFlipped={isCardFlipped}>
+        <CardFront>
+          <CardTitle>{props.name}</CardTitle>
+          <CardSubtitle>{props.city}</CardSubtitle>
+        </CardFront>
+        <CardBack>
+          <CardBody>
```

Comment:
Please get rid of the `br` elements by using something like a `grid`.

Reply:
Solved.